lib/ansible/modules/cloud/ovirt/ovirt_auth.py

```diff
@@ -221,17 +238,23 @@ def main():
     elif state == 'absent':
         params = module.params['ovirt_auth']
 
+    url = params.get('url') or os.environ.get('OVIRT_URL')
+    username = params.get('username') or os.environ.get('OVIRT_USERNAME')
+    password = params.get('password') or os.environ.get('OVIRT_PASSWORD')
+    ca_file = params.get('ca_file') or os.environ.get('OVIRT_CAFILE')
+    insecure = params.get('insecure') or ca_file is None
```

Comment:
Because the user needs to be able to force `false` in module params and have that *not* get overridden by an environment variable. You need to have a check like:

`insecure = params.get('insecure') if params.get('insecure') is not None else bool(ca_file)`

So that way a user can supply a CA file but also ask for insecure transit.